lib/fluent/plugin/gcloud_pubsub/metrics.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
Something super weird has happened to your commits here. This file already exists with this content in it plus more. 🤔

Reply:
Ah ignore me. It's Github screwing up the ordering of the commits.

Reply:
Ah I think this is probably GitHub doing its usual trick of showing commits in date order rather than actual commit order (as I moved some commits around in the history when rebasing)